jenkins/bootstrap_test.py

```diff
@@ -1265,10 +1265,11 @@ def Check(job, name):
     def testBootstrapCIYaml(self):
         # TODO(krzyzacy): temp until more jobs to be converted
         whitelist = [
-            'kubernetes-e2e-cri-gce'
+            'kubernetes-e2e-cri-gce',
+            'kubernetes-e2e-gci-gce[0-9a-z\-]+release-1.[345]'
```

Comment:
Just a way to keep track of things since I'm doing ~10 job at a time... In the end it will be all true!